src/test/java/org/cactoos/iterable/SkippedTest.java

```diff
@@ -120,19 +124,18 @@ public void skippedMoreThanExists() {
     @Test
     @SuppressWarnings("PMD.AvoidDuplicateLiterals")
```

Comment:
@vzurauskas you can remove this now I suppose

Reply:
@victornoel Removed.